n3fit/src/n3fit/layers/msr_normalization.py

```diff
@@ -61,6 +62,8 @@ def __init__(self, mode: str = "ALL", replicas: int = 1, **kwargs):
             self._msr_enabled = True
         elif mode.upper() == "VSR":
             self._vsr_enabled = True
+        elif mode.upper() == "TSR":
+            self._tsr_enabled = True
```

Comment:
I think mode.upper() == "ALL" should also activate _tsr for consistency.

Otherwise adding an ALL_polarized might make sense.

Reply:
Actually, the other sum rules except for $T_3$ and $T_8$ must not be enforced in the polarised fits as they are no longer applicable. I've removed the other sum rules when running `vp-comparefits --use_polarized` as that was very confusing.

Reply:
ah! Ok, then that's what the bug :P

None of them are?

Reply:
Unfortunately, I haven't found "the" bugs yet (if they are present) :/

But currently, I have added a check here:
https://github.com/NNPDF/nnpdf/blob/7ee6897d2aec206a69d86fc6f152abd2b26d1f3c/n3fit/src/n3fit/checks.py#L71-L80
that for polarised fits, the `sum_rules` must always be (and ony) `TSR`.